DirectEmployers/MyJobs
# PD-3381: Persist Timeline state across tabs

## What's this PR do?
Makes it to where the Timeline visibility is consistent across all tabs.

## Where should the reviewer start?
`initial-state.js`, `TabsContainer.js`, `ChartContainer.js`, then the rest of the files.

## How should this be manually tested?
1. Beta -> Analytics -> 
2. Drill down in the chart
3. Open Timeline in any of the tabs
4. Check all other tabs - Timeline should be visible and be showing the correct data.

## Any background context you want to provide?
Also, I formatted a few things (aligned component props vertically in a couple of places) so that they would fit on my screen without having to scroll horizontally. If you guys don't like that, I can revert it to how it was.
